docs/agent-template.md

```
@@ -85,16 +85,16 @@ Brief paragraph (2-3 sentences) explaining what this workflow orchestrates and i
-|-------------|---------------|-----------|
-| skill-name  | workflow-name | skill-name |
-| None        | workflow-name | skill-name |
+| Predecessor | This Agent | Successor |
+|-------------|------------|-----------|
+| skill-name  | agent-name | skill-name |
+| None        | agent-name | skill-name |
 
 ## Output Style
 
 **Expected output format:**
 ```markdown
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)